### ISSUE TITLE
database method for items

### ISSUE THREAD

**Issue by NotWes** [NotWes/Growconomy#1]

**Comment by MatteuGT** [comment]
create a test db?

**Comment by NotWes** [comment]
yeah but we need a method to automate new items.

**Comment by MatteuGT** [comment]
Where are you automating it from?

**Comment by NotWes** [comment]
Not sure.. Thats why this issue is posted.

**Comment by MatteuGT** [comment]
Yeah but we need to know what the format of the source is to determine the method to use or script to create

**Comment by NotWes** [comment]
you mean the actual database for the items?
its here: [https://github.com/NotWes/Growconomy/blob/master/database/migrations/2018_10_26_223348_create_items_table.php](url)

**Comment by MatteuGT** [comment]
Can you send a picture of the db manager page?

**Comment by NotWes** [comment]
[here](http://prntscr.com/m1kbxk)

**Comment by MatteuGT** [comment]
okay thanks. lemme take a look at it for a bit. BRB

**Comment by MatteuGT** [comment]
Well just explode it with '|' being the separator. maybe convert it to CSV first then to SQL commands?
.txt => .csv => .sql

**Comment by NotWes** [comment]
We know how to put it in the database. We just need to work out where were going to get the items from

**Comment by MatteuGT** [comment]
Oh I see, I can't help you with that, sorry 🙁
